CHANGELOG.md

```diff
@@ -11,18 +11,29 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ## Unreleased
 
+## [v0.11.0-rc.0](https://github.com/thanos-io/thanos/releases/tag/v0.11.0-rc.0) - 2020.02.19
+
 ### Fixed
 
-- [#2033](https://github.com/thanos-io/thanos/pull/2033) minio-go: Fixed Issue #1494 support Web Identity providers for IAM credentials for AWS EKS
-- [#1985](https://github.com/thanos-io/thanos/pull/1985) store gateway: Fixed case where series entry is larger than 64KB in index.
-- [#2051](https://github.com/thanos-io/thanos/pull/2051) ruler: Fixed issue where ruler does not expose shipper metrics.
-- [#2101](https://github.com/thanos-io/thanos/pull/2101) ruler: Fixed bug where thanos_alert_sender_errors_total was not registered.
+- [#2033](https://github.com/thanos-io/thanos/pull/2033) Minio-go: Fixed Issue #1494 support Web Identity providers for IAM credentials for AWS EKS
+- [#1985](https://github.com/thanos-io/thanos/pull/1985) Store Gateway: Fixed case where series entry is larger than 64KB in index.
+- [#2051](https://github.com/thanos-io/thanos/pull/2051) Ruler: Fixed issue where ruler does not expose shipper metrics.
+- [#2101](https://github.com/thanos-io/thanos/pull/2101) Ruler: Fixed bug where thanos_alert_sender_errors_total was not registered.
+- [#1789](https://github.com/thanos-io/thanos/pull/1789) Store Gateway: Improve timeouts
+- [#2009](https://github.com/thanos-io/thanos/pull/2009) Store Gateway: Add consistency delay to fetch blocks
+- [#2139](https://github.com/thanos-io/thanos/pull/2139) Properly handle SIGHUP for reloading
+- [#2040](https://github.com/thanos-io/thanos/pull/2040) UI: Fix ruler alerts click
+- [#2033](https://github.com/thanos-io/thanos/pull/1978) Fix tracing for alerts
 
 ### Added
 
+- [#2003](https://github.com/thanos-io/thanos/pull/2003) Query: Support downsampling for /series
+- [#1952](https://github.com/thanos-io/thanos/pull/1952) Store Gateway: Implement binary index header
```

Comment:
We need to elaborate here much more.. I will propose some text tomorrow. We can do that in next days for proper release as well..

Reply:
For example:

```
[#1952](https://github.com/thanos-io/thanos/pull/1952) Store Gateway: Implemented [binary index header](https://thanos.io/proposals/201912_thanos_binary_index_header.md/). This significantly reduces resource consumption (memory, CPU, net bandwidth) for startup and data loading processes as well as baseline memory. This means that adding more blocks into object storage, without querying them will use almost no resources. This, however, **still means that querying large amounts of data** will result in high spikes of memory and CPU use as before, due to simply fetching large amounts of metrics data. Since we fixed baseline, we are now focusing on query performance optimizations in separate initiatives. To enable experimental `index-header` mode run store with hidden `experimental.enable-index-header` flag.
```